### PR TITLE
New doc yulei

### PR DESCRIPTION
**Pull request by AvencastF**
new version with #135

### CODE REVIEW

File: docs/pages/advanced_example.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+
+---
+
+## Example Breakdown
+
+The process is split into two main sections:
+
+### 1. **Machine Learning**:
+   - **Files**: 
+     - [`model.py`](https://github.com/FAIR-Universe/HEP-Challenge/blob/master/sample_code_submission/model.py)
```
Comment (Ragansu) [inline docs/pages/advanced_example.md:30]: these links are outdated I would recomend 
```suggestion
     - [`model.py`](/simple_one_syst_model/model.py)
```

File: docs/pages/advanced_example.md [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+     - [`boosted_decision_tree.py`](https://github.com/FAIR-Universe/HEP-Challenge/blob/master/sample_code_submission/statistical_analysis.py)
```
Comment (Ragansu) [inline docs/pages/advanced_example.md:31]: same here
```suggestion
     - [`boosted_decision_tree.py`](/simple_one_syst_model/statistical_analysis.py)
```

File: docs/pages/advanced_example.md [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+   - Objective: Train a classifier to distinguish between [signal events](https://fair-universe.lbl.gov/docs/pages/data.html#higgs-signal) and [background events](https://fair-universe.lbl.gov/docs/pages/data.html#z-boson-background).
+
+### 2. **Statistical Analysis**:
+   - **File**: [`statistical_analysis.py`](https://github.com/FAIR-Universe/HEP-Challenge/blob/master/sample_code_submission/statistical_analysis.py)
+   - Objective: Use the classifier output to extract the signal strength through statistical analysis.
+
+> ⚠️ **Note**: 
+> **You can find all related files in the [`sample_code_submission`](https://github.com/FAIR-Universe/HEP-Challenge/tree/master/sample_code_submission)**.
```
Comment (Ragansu) [inline docs/pages/advanced_example.md:39]: same here
```suggestion
> **You can find all related files in the [`sample_code_submission`](https://github.com/FAIR-Universe/HEP-Challenge/tree/master/simple_one_syst_model)**.
```

**Review by Ragansu — CHANGES_REQUESTED** [review]
There is some places were the old name is use `sample_code_submission` which was replaced by `simple_one_syst_model` for clarity. and for links I change them to internal links so it works within a text editor and github.

**Comment by Ragansu** [comment]
Also can You change the check if I missed any renaming.

**Review by Ragansu — COMMENTED** [review]
Looks Good.

**Review by Ragansu — APPROVED** [review]

**Comment by dhrou** [comment]
Looks good to me. Ragansu, to save time,  could you use this PR to fix data.md : the table with preselection cut, keep only pt had and pt jet, the others can be confusing (because they do not change). (maybe write somewhere as text that "events are preselected to have one lepton one tau hadron of opposite charge").

**Comment by Ragansu** [comment]
Its easier for me make another PR since this is on the fork of Yulei. 
I am making another PR to clean up the documentation a bit. So I will do it there.

**Pull request merged**